docs/changelog/0.7.0.rst

```diff
@@ -235,7 +235,7 @@ Improved Documentation
 - Updated the |documentation guide| to include a description on how to
   add and cite references to PlasmaPy's global bibliography BibTeX_ file,
   :file:`docs/bibliography.bib`. (`#1263 <https://github.com/plasmapy/plasmapy/pull/1263>`__)
-- Added sphinxcontrib-bibtex_ as a Sphinx_ extension to enable references
+- Added ``sphinxcontrib-bibtex`` as a Sphinx_ extension to enable references
```

Comment:
Most of these substitutions were used only in the documentation guide and a changelog entry.  It's not important to have these links in the changelog entries, so I'm converting them to in-line code-snippets.